src/TConsole.cpp

```diff
@@ -40,6 +40,7 @@
 #include "pre_guard.h"
 #include <QLineEdit>
 #include <QMessageBox>
+#include <QMimeData>
```

Comment:
As it happens I happen to have this same fix in #3515 (because it hit my work as well) - but that is not a problems as it will disappear from here when I merge that one in - and this change will be *fast-forwarded* over without a problem. :pray: